python/hsfs/feature_view.py

```diff
@@ -219,10 +219,10 @@ def get_feature_vectors(
 
     def get_batch_data(self, start_time=None, end_time=None, read_options=None):
         """
-        start_time: timestamp in second or wallclock_time: Datetime string. The String should be formatted in one of the
-            following formats `%Y%m%d`, `%Y%m%d%H`, `%Y%m%d%H%M`, `%Y%m%d%H%M%S`, or `%Y%m%d%H%M%S%f`.
-        end_time: timestamp in second or wallclock_time: Datetime string. The String should be formatted in one of the
-            following formats `%Y%m%d`, `%Y%m%d%H`, `%Y%m%d%H%M`, `%Y%m%d%H%M%S`,  or `%Y%m%d%H%M%S%f`.
+        start_time: timestamp in second or wallclock_time: Datetime object or string. The String should be formatted in
```

Comment:
I don't see a call of the timestamp conversion in the call trace of 

fv.get_batch_data()->fv-engine.get_batch_data()->fv-engine.get_batch_query()->fv-api->get_batch_query()